pkg/krmtotf/references_test.go

```diff
@@ -385,6 +386,10 @@ func TestResolveResourceReferenceToTFResource(t *testing.T) {
 					},
 				},
 			},
+			expectedFinalConfig: map[string]interface{}{
```

Comment:
The error here in this test is "resource not ready"(but exist), not "resource not found". I mention the expectedFinalConfig here to distinct it from the "resource not found" test. It would be preferable for us to verify the specific error message though.

Reply:
Gotcha. My concern is that in the test logic, there is no validation of this field, so it can be changed into any value (valid/invalid) without being detected easily.
Are you suggesting that the existence of this field helps readability no matter the value is correct or not? If so, we can leave it as is. If not, then I think we should at least add a TODO to ensure the the value of `expectedFinalConfig` is checked for this test case in the test logic, because accidental changes of this field may harm readability.